packages/shared/src/variants/s_fractional.ts

```diff
@@ -308,6 +313,32 @@ export class SFractional extends AbstractGame<
       secondary_colors: ["#0000FF", "#FF0000"],
     };
   }
+
+  static uiTransform(
+    config: SFractionalConfig,
+    gamestate: SFractionalState,
+  ): {
+    config: SFractionalConfig & DefaultBoardConfig;
```

Comment:
I think you can just type this as `SFractionalConfig` and TypeScript will understand it satisfies `DefaultBoardConfig`.  But it's fine to keep it this way too.

Reply:
Makes sense!